ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewBackgroundDrawable.java

```diff
@@ -298,7 +300,9 @@ private void updatePath() {
         bottomLeftRadius + extraRadiusForOutline
       },
       Path.Direction.CW);
+  }
 
+  private void updatePathEffectForBorderStyle() {
```

Comment:
Can you add a short comment explaining what this does? Is it to draw e.g. dashed or dotted borders?